robert_beverage/package.xml

```diff
@@ -11,4 +11,5 @@
   <buildtool_depend>catkin</buildtool_depend>
   <run_depend>rospy</run_depend>
   <run_depend>python-qrencode</run_depend>
+  <run_depend>tf2_tools</run_depend>
```

Comment:
why do you need this package?

Reply:
why do you need this package?

Reply:
I used it to build a frame map for better understanding. Do you want me to delete it?

Reply:
yes, please. There is already [the following](http://wiki.ros.org/rqt_tf_tree) RQT plugin installed on your Docker image
You could start it by running
`rosrun rqt_tf_tree rqt_tf_tree`